recipes/iconfontcppheaders/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,8 @@
+#include <IconFontCppHeaders/IconsFontAwesome6.h>
```

Comment:
```suggestion
#include <IconFontCppHeaders/IconsFontAwesome6.h>
```
Related with the comment I made about `copy(self, "*.h", self.source_folder, os.path.join(self.package_folder, "include", "IconFontCppHeaders"))` but I'm not completely sure about that